kong/plugins/zipkin/handler.lua

```diff
@@ -131,6 +131,14 @@ if subsystem == "http" then
     request_span:set_tag("http.method", method)
     request_span:set_tag("http.path", req.get_path())
 
+    local static_tags = conf.static_tags
+    if type(static_tags) == "table" then
```

Comment:
What other types this value could it be? Will there be an earlier sign of failure if the value is set to the wrong type?